Moblin/Various/Model/ModelKick.swift

```diff
@@ -245,4 +254,21 @@ extension Model: KickOusherDelegate {
             )
         }
     }
+
+    func kickPusherFetchSubscriberBadges(completion: @escaping ([SubscriberBadge]) -> Void) {
```

Comment:
Just pass the channel name to the kick pusher initializer and let the pusher fetch the badges itself. no need to involve the model, right?